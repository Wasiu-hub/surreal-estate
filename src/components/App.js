/* eslint-disable react/function-component-definition */
/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable react/jsx-filename-extension */
/* eslint-disable quotes */
import "../styles/App.css";

function App() {
  return <h2>Surreal Estate</h2>;
}

export default App;
