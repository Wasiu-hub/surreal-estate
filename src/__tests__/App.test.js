/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
/* eslint-disable quotes */
import { render, screen } from "@testing-library/react";
import App from "../components/App";

test("renders learn react link", () => {
  render(<App />);
  const linkElement = screen.getByText(/Surreal Estate/i);
  expect(linkElement).toBeInTheDocument();
});
