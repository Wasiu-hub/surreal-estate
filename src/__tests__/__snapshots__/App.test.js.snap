// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="navbar"
    >
      <img
        alt="estate-logo"
        class="logo"
        src="logo.png"
      />
      <ul
        class="navbar-links"
      >
        <li
          class="navbar-links-item"
        >
          <a
            class="item"
            href="/"
          >
            View Properties
          </a>
        </li>
        <li
          class="navbar-links-item"
        >
          <a
            class="item"
            href="/add-property"
          >
            Add a Property
          </a>
        </li>
      </ul>
    </div>
    <div>
      Properties Page
    </div>
  </div>
</DocumentFragment>
`;
